jwala-services/src/main/java/com/cerner/jwala/service/jvm/state/JvmStateReceiverAdapter.java

```diff
@@ -69,18 +70,73 @@ public void receive(final Message jGroupMsg) {
         }
     }
 
+    private JvmState getJvmState(final Map serverInfoMap) {
+
+        // first check if the key types are string
+        final Set keys = serverInfoMap.keySet();
+        if (keys.isEmpty()) {
+            return null;
+        }
+
+        final Object initialKey = keys.iterator().next();
+        if (initialKey instanceof String) {
```

Comment:
I think we can use this instead

    if (keys.contain(STATE_KEY)) {
        ...
    }

We won't have an invalid type error since contains uses the hash code to look for the key and I think the code would be easier to understand. I think I might have written this code somewhere so it's my bad if I did.